tools/tesseratos/CMakeLists.txt

```diff
@@ -22,12 +20,7 @@ set(TESSERATOS_SOURCE
 	"src/tesseratos/ecs_statistics/plugin.cpp"
 )
 
-add_library(tesseratos ${TESSERATOS_SOURCE})
-target_include_directories(tesseratos PUBLIC "include")
```

Comment:
Shouldn't we want the individual tools to be able to be added to the game itself? A lot of games have a debug mode that toggles a lot of developer tools. It might be useful for devs if they can test stuff in the build itself while still having access to some of the tools.

Reply:
Agree, I think we should gradually move stuff to maybe a `engine/tools` directory, but that would be a big change to include in this PR. We need to decide which of tools we want to include.